src/Symfony/Component/Security/Core/composer.json

```diff
@@ -26,13 +26,15 @@
         "symfony/translation": "~2.0,>=2.0.5|~3.0.0",
         "symfony/validator": "~2.5,>=2.5.5|~3.0.0",
         "psr/log": "~1.0",
-        "ircmaxell/password-compat": "1.0.*"
+        "ircmaxell/password-compat": "1.0.*",
+        "symfony/ldap": "~2.8|~3.0.0"
```

Comment:
Sorry, read too fast, it's only for required dev. I'm gonna delete both comments.